django/contrib/sitemaps/management/commands/ping_google.py

```diff
@@ -7,6 +7,10 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('sitemap_url', nargs='?')
+        parser.add_argument('--sitemap-uses-http', action='store_true')
 
     def handle(self, *args, **options):
-        ping_google(sitemap_url=options['sitemap_url'])
+        ping_google(
+            sitemap_url=options['sitemap_url'],
+            sitemap_uses_https=not options['sitemap_uses_http'],
```

Comment:
I nearly missed the `not` before `options['sitemap_uses_http']` while reading.

Reply:
I guess the `add_argument` could be `action='store_false', dest='use_https'`

Reply:
Yeah, that would certainly be a better idea, rather than using `not` here for negation.

Reply:
I find argument parsing logic more complicated than a boolean. We could add an intermediate variable:
```
sitemap_uses_https = not options['sitemap_uses_http']
```
if you think readability is poor with the current version but I don't predict much chance of confusion.

Reply:
Yup, this extra line won't hurt, IMHO.

Reply:
Would you change remove this pattern in all of Django?
```
django/db/models/fields/related.py:            m2m=not self.unique,
django/db/models/fields/related.py:            m2m=not self.unique,
django/core/management/commands/testserver.py:        db_name = connection.creation.create_test_db(verbosity=verbosity, autoclobber=not interactive, serialize=False)
django/contrib/sessions/backends/db.py:                obj.save(force_insert=must_create, force_update=not must_create, using=using)
django/contrib/postgres/search.py:        return type(self)(self.value, config=self.config, invert=not self.invert)
django/contrib/contenttypes/fields.py:            m2m=not self.unique,
django/contrib/admin/options.py:        ModelForm = self.get_form(request, obj, change=not add)
django/contrib/admin/options.py:                new_object = self.save_form(request, form, change=not add)
django/contrib/admin/options.py:            formsets, inline_instances = self._create_formsets(request, new_object, change=not add)
django/contrib/admin/options.py:        return self.render_change_form(request, context, add=add, change=not add, obj=obj, form_url=form_url)
django/test/utils.py:                    autoclobber=not interactive,
```
I don't find it problematic, especially because my text editor bolds `not` so it's difficult to overlook.